docs/user-settings.md

```diff
@@ -155,6 +155,18 @@ language: {
 }
 ```
 
+#### webcomponent
+
+- **type** object (optional)
```

Comment:
I tested it and it supports also boolean like webcomponent:true 
I think we can also add `OR boolean`